lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -274,8 +274,8 @@ function _campaign_resource_signup($nid, $values) {
     unset($values['northstar_id']);
   }
 
-  // If we pass either a Northstar ID or Drupal UID to `uid`, convert it to a real UID.
-  $values['uid'] = dosomething_user_convert_to_legacy_id($values['uid']);
+  // If a Drupal UID is passed to `uid`, convert it to the user's Northstar ID.
+  $values['uid'] = dosomething_user_convert_to_northstar_id($values['uid']);
```

Comment:
Ah that totally makes sense, thanks for clarifying, @DFurnes ! So then I think you're right - when going through the code as far as I could tell, the only thing that still rely on absolutely having more info. about the user (that is stored in P-A) is for sending to third party. I'll make a note in the card and let @ngjo know that this is blocked until we are able to send notifications via Blink/Customer.io.